webapp/src/actions.js

```diff
@@ -0,0 +1,21 @@
+import * as ActionTypes from './action_types';
+
+export function connected(client) {
```

Comment:
It's for kind-of-consistency in case it is ever changed to dispatch an action. I can't find the PR, but we had a similar discussion somewhere else about how doing this sort of thing is a bit weird, but I think we decided to leave it this way.

I also like that it keeps client calls encapsulated in this part of the code, although that's more of a personal thing